packs/autoscale/actions/workflows/asg_delete_node.yaml

```diff
@@ -87,8 +79,6 @@ workflows:
           - failure_get_dns_zone_id
       get_dns_record_id:
         action: rackspace.find_dns_record_id
-        policies:
-          wait-before: 1
         input:
           zone_id: <% $.dns_zone_id %>
```

Comment:
here's a new handy feature of referencing tasks outputs without publishing: 

`<% $.get_dns_zone_id.result %>`